softdevice_controller/include/sdc_hci_vs.h

```diff
@@ -1122,6 +1171,41 @@ uint8_t sdc_hci_cmd_vs_qos_channel_survey_enable(const sdc_hci_cmd_vs_qos_channe
  */
 uint8_t sdc_hci_cmd_vs_set_power_control_apr_handling(const sdc_hci_cmd_vs_set_power_control_apr_handling_t * p_params);
 
+/** @brief Set LE Power Control Request procedure parameters.
+ *
+ * This command sets the parameters used in LE Power Control Request
+ * procedure by the Link Layer.
+ *
+ * beta parameter is used for exponential weighted averaging of RSSI.
+ * The average RSSI is calculated using the following formula,
+ * avg[n] = beta * avg[n - 1] + (1 - beta) * rssi[n]
+ *
+ * Average RSSI and lower_limit parameter are used to calculate APR value the controller sends
+ * in LL_POWER_CONTROL_RSP.
+ *
+ * When auto_enable parameter is set, the controller will keep average RSSI within
+ * [lower_limit, upper_limit] bounds. When the average RSSI goes out of these bounds, the
+ * controller will autonomously send LL_POWER_CONTROL_REQ requesting to adjust the
+ * peer's TX power so average RSSI becomes either lower_target_rssi or upper_target_rssi.
+ * The controller will not send such requests more often than specified by wait_period_ms parameter.
+ *
+ * When apr_enable parameter is set, the controller will adjust local TX power according to
+ * APR value received from the peer in LL_POWER_CONTROL_RSP and the apr_margin parameter.
+ *
+ * When this command is issued, the controller stores the parameters and
+ * uses them for the subsequent LE Power Control Request procedures across all the connections.
+ *
+ * Event(s) generated (unless masked away):
+ * When the command has completed, an HCI_Command_Complete event shall be generated.
+ *
+ * @param[in]  p_params Input parameters.
+ *
+ * @retval 0 if success.
+ * @return Returns value between 0x01-0xFF in case of error.
+ *         See Vol 2, Part D, Error for a list of error codes and descriptions.
+ */
+uint8_t sdc_hci_cmd_vs_set_power_control_request_params(const sdc_hci_cmd_vs_set_power_control_request_params_t * p_params);
```

Comment:
Is this something we should mention in the changelog?

Reply:
@seko-nordic , it was intentional not mention anything now, right?

Reply:
Yes this was on purpose. Reason being that it is not completely implemented, but once it is, it will replace the two existing VS commands sdc_hci_cmd_vs_set_auto_power_control_request_param and sdc_hci_cmd_vs_set_power_control_apr_handling. We want to minimize confusion with this, as customers cannot use it yet. (Answering as Sergey is in vacation at the moment.)